src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py

```diff
@@ -215,6 +215,9 @@ def container_services(self):
         elif api_version == '2020-03-01':
             from .v2019_04_01.operations import ContainerServicesOperations as OperationClass
             api_version = "2019-04-01" #2020-03-01 generated Python SDK contains no list_orchestrator op
+        elif api_version == '2020-04-01':
+            from .v2019_04_01.operations import ContainerServicesOperations as OperationClass
+            api_version = "2019-04-01" #2020-04-01 generated Python SDK contains no list_orchestrator op
```

Comment:
also need to update setup.py and version.py to generate a new version cli-extension.
update HISTORY.md